Library/Homebrew/formula.rb

```diff
@@ -475,8 +473,9 @@ def inspect
   # less consistent and the standard parameters are more memorable.
   def std_cmake_args
     %W[
+      -DCMAKE_C_FLAGS_RELEASE=""
```

Comment:
Cool. I've built a few popular things against this without any errors yet, but I'll give it a wider run through this week.